src/domain/community/community/VirtualContributorsBlock/VirtualContributorsDialog.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { DialogContent, ListItemButton, ListItemButtonProps, ListItemButtonTypeMap } from '@mui/material';
+import DialogWithGrid from '../../../../core/ui/dialog/DialogWithGrid';
+import DialogHeader from '../../../../core/ui/dialog/DialogHeader';
+import Gutters from '../../../../core/ui/grid/Gutters';
+import BadgeCardView from '../../../../core/ui/list/BadgeCardView';
+import Avatar from '../../../../core/ui/avatar/Avatar';
+import RouterLink, { RouterLinkProps } from '../../../../core/ui/link/RouterLink';
+import { BlockSectionTitle } from '../../../../core/ui/typography';
+import { SearchVisibility } from '../../../../core/apollo/generated/graphql-schema';
+import SearchField from '../../../../core/ui/search/SearchField';
+import VCIcon from '../../virtualContributor/VirtualContributorsIcons';
+
+export interface VirtualContributorProps {
+  id: string;
+  searchVisibility: SearchVisibility;
+  profile: {
+    displayName: string;
+    tagline: string;
+    avatar?: {
+      uri: string;
+    };
+    url: string;
+  };
+}
+
+interface VirtualContributorsDialogProps {
+  open: boolean;
+  onClose: () => void;
+  virtualContributors: VirtualContributorProps[];
+}
+
+const VirtualContributorsDialog: FC<VirtualContributorsDialogProps> = ({ open, onClose, virtualContributors }) => {
+  const { t } = useTranslation();
+
+  const Wrapper = <D extends React.ElementType = ListItemButtonTypeMap['defaultComponent'], P = {}>(
+    props: ListItemButtonProps<D, P> & RouterLinkProps
+  ) => <ListItemButton component={RouterLink} {...props} />;
+
+  const [filter, setFilter] = useState<string>('');
+  const filterVCs = (virtualContributor: VirtualContributorProps) => {
+    const lowerCaseFilter = filter.toLowerCase();
+
+    return (
+      virtualContributor.profile.displayName.toLowerCase().includes(lowerCaseFilter) ||
+      virtualContributor.profile.tagline.toLowerCase().includes(lowerCaseFilter)
```

Comment:
Why are you searching in the tagline? I think it should be only on `displayName` (unless specified otherwise).